tests/catalog/test_glue.py

```diff
@@ -46,6 +47,23 @@
 )
 
 
+@mock_aws
+def test_load_catalog_from_impl() -> None:
+    assert isinstance(
+        load_catalog(
+            "catalog",
+            **{
+                PY_CATALOG_IMPL: "pyiceberg.catalog.glue.GlueCatalog",
+                "region_name": "us-east-1",
+                "aws_access_key_id": "access-key",
+                "aws_secret_access_key": "secret-key",
+                "aws_session_token": "session-token",
```

Comment:
These property names will be deprecated in the upcoming release: https://github.com/apache/iceberg-python/pull/922

How about using the new `glue.` config names here:
https://github.com/apache/iceberg-python/blob/d0bfb4a67bccd15c2d2e9baa481a5f33fe4c5220/mkdocs/docs/configuration.md?plain=1#L312-L315

Reply:
I moved to use SqlCatalog